.github/workflows/android-feature.yml

```diff
@@ -16,7 +16,7 @@ jobs:
 
     strategy:
       matrix:
-        module: [ ':sdk:core', ':sdk:analytics', ':sdk:forms', ':sdk:push-fcm' ]
+        module: [ ':sdk:core', ':sdk:analytics', ':sdk:forms', ':sdk:location', ':sdk:push-fcm' ]
```

Comment:
Tests from this module now running in CI
<img width="819" height="46" alt="Screenshot 2025-09-09 at 13 31 33" src="https://github.com/user-attachments/assets/bd2773c9-069a-4803-9d4a-07fce52cc947" />